spec/basic/render_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "spec_helper"
```

Comment:
I would put a `# frozen_string_literal: true` here.

Reply:
About time i figured out what this does 👍